### PR TITLE
Add new training materials request to nfdi4bioimage.yml

### PR DESCRIPTION
**Pull request by lea-33**
Added new training materials.

### CODE REVIEW

**Comment by lea-33** [comment]
this is a REAL NEW entry for the yml file (not only for testing). It worked now!

File: resources/nfdi4bioimage.yml [hunk 1]
```diff
@@ -4483,4 +4483,19 @@ resources:
   url:
   - https://zenodo.org/records/13331351
   - https://doi.org/10.5281/zenodo.13331351
+  
+- authors: ''
+  license:
+  - CC-BY-SA-4.0
+  - CC-BY-NC-4.0
```
Comment (haesleinhuepf) [inline resources/nfdi4bioimage.yml:4490]:
```suggestion
  - CC-BY-NC-SA-4.0
```
I quickly checked the footer of the website. It says this license

File: resources/nfdi4bioimage.yml [hunk 2]
```diff
@@ -4483,4 +4483,19 @@ resources:
   url:
   - https://zenodo.org/records/13331351
   - https://doi.org/10.5281/zenodo.13331351
+  
+- authors: ''
```
Comment (haesleinhuepf) [inline resources/nfdi4bioimage.yml:4487]:
```suggestion
- authors: 'Rémy Jean Daniel Dornier'
```

There is a box "Last edited by" which says this name. Maybe, there are more authors considering former versions....

**Review by haesleinhuepf — CHANGES_REQUESTED** [review]
Just two minor things: I found it suspicious that there are no authors... also the license might be better specified like this.

**Pull request merged**